docs/userguide/serialization.md

```diff
@@ -0,0 +1,390 @@
+    "values": ["low", "high"]
+}
+"""
+task_parameter_reconstructed = Parameter.from_json(task_parameter_str)
+assert task_parameter == task_parameter_reconstructed
+```
+
+```{note} 
+When serializing an object that belongs to a class hierarchy, BayBE automatically
+injects the `type` field into the serialization string to enable frictionless deserialization
+at a later stage.
+```
+
+### Using abbreviations
+Classes that have an `abbreviation` class variable defined can be conveniently
+deserialization using the corresponding abbreviation string:
+```python
+from baybe.acquisition.base import AcquisitionFunction
+
+acqf1 = AcquisitionFunction.from_json('{"type": "UpperConfidenceBound"}')
+acqf2 = AcquisitionFunction.from_json('{"type": "UCB"}')
+
+assert acqf1 == acqf2
+```
+
+(NESTED_OBJECTS)=
+### Nesting objects
+BayBE objects typically appear as part of a larger object hierarchy.
+For instance, a
+{class}`SearchSpace <baybe.searchspace.core.SearchSpace>` can hold one or several
+{class}`Parameters <baybe.parameters.base.Parameter>`, just like an
+{class}`Objective <baybe.objectives.base.Objective>` can hold one ore several 
+{class}`Targets <baybe.targets.base.Target>`.
+This hierarchical structure can be directly replicated in the serialization string:
+
+```python
+from baybe.objectives import DesirabilityObjective
+from baybe.targets import NumericalTarget
+
+objective = DesirabilityObjective(
+    targets=[
+        NumericalTarget(name="t1", mode="MAX", bounds=(-1, 1)),
+        NumericalTarget(name="t2", mode="MIN", bounds=(0, 1)),
+    ],
+    weights=[0.1, 0.9],
+    scalarizer="MEAN",
+)
+
+objective_str = """
+{   
+    "targets": [
+        {
+            "type": "NumericalTarget",
+            "name": "t1",
+            "mode": "MAX",
+            "bounds": [-1.0, 1.0]
+        }, 
+        {
+            "type": "NumericalTarget",
+            "name": "t2",
+            "mode": "MIN",
+            "bounds": [0.0, 1.0]
+        }
+    ],
+    "weights": [0.1, 0.9],
+    "scalarizer": "MEAN"
+}
+"""
+
+assert objective == DesirabilityObjective.from_json(objective_str)
+```
+
+(ALTERNATIVE_CONSTRUCTORS)=
+### Invoking alternative constructors
+Many BayBE classes offer additional routes of construction next to the default
+mechanism via the class' `__init__` method.
+This offers convenient ways of object initialization alternative to specifying
+an object's attributes in their "canonical" form, which is often not the preferred
+approach.
+
+For instance, a search space is composed of two sub-components, a
+[discrete subspace](baybe.searchspace.discrete.SubspaceDiscrete)
+and a [continuous subspace](baybe.searchspace.continuous.SubspaceContinuous),
+which are accordingly expected by the 
+[`SearchSpace`](baybe.searchspace.core.SearchSpace.__init__) constructor.
+However, instead of providing the two components directly, most users would more
+naturally invoke one of the alternative `classmethods` available, such as
```

Comment:
Why is classmethods here in backticks? Isn't it just used as a regular word?

Reply:
removed